classes/author.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'item'
+
+class Author < Item
+  attr_accessor :id, :first_name, :last_name
+  attr_reader :items
+
+  def initialize(params)
+    super(params)
+
+    @id = params['id']
```

Comment:
It would better to add a random id because adding a random number to the id parameter can be useful in situations where you need to ensure that the id is unique. For example, if you are creating a new record in a database and you want to ensure that the id is unique, you can generate a random number and assign it to the id parameter. This helps prevent conflicts with existing records and ensures that each record has a unique identifier 😊
you can check below code 👇

**@id = params[:id] || rand(1..10_000)**